proposals/NNNN-atf-script-runner.md

```diff
@@ -0,0 +1,41 @@
+# ATF Script Runner
+
+* Proposal: [SDL-NNNN](NNNN-atf-script-runner.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+ATF is a great tool for both developers and testers that help test any change in SDL.
+But it lacks a powerful test script runner.
+
+## Motivation
+
+Currently ATF is unable to:
+
+  - run multiple test scripts
+  - run test script in parallel
+  - create test reports in standard format
+
+## Proposed solution
```

Comment:
Please provide more information on how this proposed solution would be implemented.

Reply:
@theresalech Proposal has been updated: added diagram and made some corrections.
Regarding the implementation - it's not too much can be added. Script runner is just a command line tool that can be developed using any programming language, e.g. C++, Python etc.